contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -234,13 +234,18 @@ $$SELECT
 
 
 /*
- * COLUMNS view
+ * COLUMNS view internal
  */
 
-CREATE OR REPLACE VIEW information_schema_tsql.columns AS
-	SELECT CAST(nc.dbname AS sys.nvarchar(128)) AS "TABLE_CATALOG",
+CREATE OR REPLACE VIEW information_schema_tsql.columns_internal AS
+	SELECT a.attrelid AS "TABLE_OID",
```

Comment:
IMO, we should use pg_class.oid instead of pg_attribute.attrelid. It won't affect the result but as the column name is "TABLE_OID" let us obtain it from pg_class as that makes more sense.

Reply:
Done